sycl/test/warnings/warnings.cpp

```diff
@@ -1,5 +1,5 @@
 // RUN: %clangxx -fsycl --no-system-header-prefix=CL/sycl -fsyntax-only -Wall -Wextra -Wno-ignored-attributes -Wno-deprecated-declarations -Wpessimizing-move -Wunused-variable -Wmismatched-tags -Wunneeded-internal-declaration -Werror -Wno-unknown-cuda-version %s -o %t.out
-
+// UNSUPPORTED: linux
```

Comment:
Please, revert or provide a comment similar to the comment in sycl/test/sub_group/load_store.cpp clarifying the reason for this test to be unsupported on linux.

Reply:
sycl/test/usm/smem_varied.cpp and sycl/test/warnings/warnings.cpp also hang with GPU driver. But currently GPU RT in share folder was deleted, we need to wait GPU RT restore and verify again.

Reply:
sycl/test/warnings/warnings.cpp only compiles on the host, it doesn't use GPU at all, so I don't think it might cause any hangs on GPU.
Could you check that you have functional environment on your system, please?

Reply:
Got it